portal/server/routes/contract/dealWithFacilityFlags.js

```diff
@@ -0,0 +1,30 @@
+const canAmendFacility = require('./canAmendFacility');
```

Comment:
This file is renamed from `dealWithCanIssueOrEditIssueFacilityFlags.js`,

I've also changed lines 10-25